ws4redis/websocket.py

```diff
@@ -198,14 +198,14 @@ def receive(self):
             raise WebSocketError("Connection is already closed")
         try:
             return self.read_message()
-        except UnicodeError:
-            logger.info('websocket.receive: UnicodeError')
+        except UnicodeError as e:
+            logger.info(u'websocket.receive: UnicodeError {}'.format(e))
             self.close(1007)
-        except WebSocketError:
-            logger.info('websocket.receive: WebSocketError')
+        except WebSocketError as e:
+            logger.info(u'websocket.receive: WebSocketError {}'.format(e))
             self.close(1002)
         except Exception as e:
-            logger.info('websocket.receive: Unknown error %s', e)
+            logger.info(u'websocket.receive: Unknown error {}'.format(e))
```

Comment:
Also, I noticed that all the CI builds have failed for Django1.7 (which
they only started testing with build #130).  Seems like the fixtures don't
work.  Are you working on fixing this or "get around to it sometime"? (Cuz
I we are porting our app to 1.7 in the next month...)

Cheers,
 Ethan

2015-04-23 18:20 GMT+02:00 Ethan Snyder-Frey <ethan@orderbird.com>:

> I made the fixes in commit 969e702.  Maybe this is now safe for merging?
> If you don't like them, you can also ignore these changes, but they were
> nice in local debugging.
>
> The most important ones in my eyes where the redis_cleanup, which was
> already merged, and the nginx_safe_response.  As we had some interesting
> hanging connections in a production environment when I raised
> PermissionDenied.
>
> By the way, this is a pretty awesome package.  In one day I got the
> websocket server written and unit tested.  Production loadtesting and
> playing well with the clients took a bit longer though.  I replaced a
> unmaintable homebrew tornado app with this.
>
> Cheers,
>  Ethan
>
>
>
> 2015-04-23 17:52 GMT+02:00 Jacob Rief <notifications@github.com>:
>
>> In ws4redis/websocket.py
>> <https://github.com/jrief/django-websocket-redis/pull/94#discussion_r28975770>
>> :
>>
>> >              self.close(1002)
>> >          except Exception as e:
>> > -            logger.info('websocket.receive: Unknown error %s', e)
>> > +            logger.info(u'websocket.receive: Unknown error {}'.format(e))
>>
>> Just a note. I pilfered this file, therefore the old string formatting
>> was used. Is this file now Py3 safe?
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/jrief/django-websocket-redis/pull/94/files#r28975770>
>> .
>>
>
>